clusters_calculator.py

```diff
@@ -6,16 +6,20 @@
 import multiprocessing
 
 
-def retrieve_clusters(ne_lat, ne_lng, sw_lat, sw_lng, start_date, end_date, fatal, severe, light, inaccurate, zoom):
-    marker_boxes = divide_to_boxes(ne_lat, ne_lng, sw_lat, sw_lng)
+def retrieve_clusters(**kwargs):
+    marker_boxes = divide_to_boxes(kwargs['ne_lat'], kwargs['ne_lng'], kwargs['sw_lat'], kwargs['sw_lng'])
     result_futures = []
     logging.info('number of cores: ' + str(multiprocessing.cpu_count()))
     with concurrent.futures.ThreadPoolExecutor(max_workers=multiprocessing.cpu_count()) as executor:
         for marker_box in marker_boxes:
-            markers_in_box = Marker.bounding_box_query(marker_box[0], marker_box[1], marker_box[2], marker_box[3],
-                                                         start_date, end_date, fatal,
-                                                         severe, light, inaccurate).all()
-            result_futures.append(executor.submit(calculate_clusters, markers_in_box, zoom))
+
+            clusters_kwargs = kwargs
```

Comment:
If you want to copy the dictionary you should call `dict(kwargs)`. Otherwise it's another reference to the same dictionary.
And why won't `divide_to_boxes` return a dictionary too, rather than a tuple? Then you can just call `clusters_kwargs.update(...)` with that dictionary and it will update the matching entries.